src/components/Home.vue

```diff
@@ -66,9 +65,6 @@
       this.reservations[1].date = this.today.add(1, 'day')
     },
     computed: {
-      hasUserData() {
```

Comment:
Removing this user data stuff, which is left over from when I first set up the app. It's not needed in the Home page now.